pkg/scheduler/plugins/nodeorder/nodeorder.go

```diff
@@ -131,8 +129,8 @@ func calculateWeight(args framework.Arguments) priorityWeight {
 func (pp *nodeOrderPlugin) OnSessionOpen(ssn *framework.Session) {
 	weight := calculateWeight(pp.pluginArguments)
 	pl := util.NewPodLister(ssn)
-	pods, _ := pl.List(labels.NewSelector())
-	nodeMap, nodeSlice := util.GenerateNodeMapAndSlice(ssn.Nodes)
+	//pods, _ := pl.List(labels.NewSelector())
+	nodeMap := util.GenerateNodeMapAndSlice(ssn.Nodes)
```

Comment:
Why remove pods?

Reply:
ssn.Nodes has all pods info , and will be updated after one task is finished. 
But if use relist pods, with gang scheduler, podAntiAffinity not take effect.In the same job, two anti-affinity PODs scheduled on the same node.
```
---
apiVersion: scheduling.volcano.sh/v1beta1
kind: PodGroup
metadata:
  name: test
  namespace: default
spec:
  minMember: 2
---
apiVersion: apps/v1
kind: Deployment
metadata:
  name: my-nginx
spec:
  selector:
    matchLabels:
      app: my-nginx
  replicas: 2
  template:
    metadata:
      labels:
        app: my-nginx
      annotations:
        scheduling.k8s.io/group-name: test
    spec:
      schedulerName: volcano
      containers:
      - name: my-nginx
        image: nginx
        ports:
        - containerPort: 80
        resources:
          requests:
            memory: "1Gi"
            cpu: "1000m"
          limits:
            memory: "1Gi"
            cpu: "1000m"
      affinity:
        podAntiAffinity:
          requiredDuringSchedulingIgnoredDuringExecution:
            - labelSelector:
                matchExpressions:
                  - key: app
                    operator: In
                    values:
                      - "my-nginx"
              topologyKey: "kubernetes.io/hostname"
```